### DISCUSSION TITLE
Privacy?

### DISCUSSION THREAD

**Discussion by bring-shrubbery** [statelyai/inspect#23] [Q&A] [answered]
Hey there,

I want to use this package in my company's codebase, but I'm a bit worried about compliance and privacy, since it runs on a different domain (not local).

Is there a write-up somewhere which data is being shared? Something like privacy policy just for this tool? Also would be great to know how it works in general.

I could dive into code and figure it out myself, but it would be great to be able to send a link to my security team and get approval from them.

P.S. Awesome product by the way :)

**Comment by davidkpiano** [comment, marked as answer]
Our privacy statement can be found here: https://stately.ai/privacy

As far as the inspector, nothing is ever stored (machine, states, events, etc.) on our database. 

You can also set the URL to your own domain and visualize the events yourself, but we don't yet provide a way to self-host the actual inspector that visualizes the state machine and states/events/sequence diagrams.

```ts
const inspector = createInspector({
  url: 'http://localhost:3000', // will never go to stately.ai
  // ...
})
```

**Reply by bring-shrubbery** [reply]
Nice, that's exactly what I've been looking for!
